### ISSUE TITLE
Gdal issue when importing raster topography

### ISSUE THREAD

**Issue by lperozzi** [cgre-aachen/gempy#568]
**Describe the bug**
I followed the installation on Mac OS X (https://docs.gempy.org/installation.html#macos-installation-guide-may-2020). All works fine except when I try to read à `.tif` using the `source='gdal'` option with the `set_topography` method, that gives me the an error saying the `Gdal package is not installed. No support for raster formats`. (I confirm that Gdal is installed through `conda install gdal`)

**Desktop (please complete the following information):**
  * OS: `OS X Catalina 10.15.7`
  * GemPy Version
    * `2.2.7`

**Additional context**

The version of gdal installed through conda is `3.2.1`

![image](https://user-images.githubusercontent.com/6198369/105673962-13f1c480-5ee7-11eb-8eac-2583c23024ee.png)

**Comment by lperozzi** [comment]
I've tried something that may works. 

In `[create_topography.py](https://github.com/cgre-aachen/gempy/blob/master/gempy/core/grid_modules/create_topography.py)` I modified line 17 from `ìmport gdal` to `from osgeo import gdal` and it seems to work now.

Is it something that can be updated for the next release?

**Comment by AlexanderJuestel** [comment]
Hello @lperozzi,

thanks for noting that. This should be updated for a next release as `import gdal` is deprecated according to the PyPi page (https://pypi.org/project/GDAL/). These are just compatibility modules and they will be deprecated. 

As suggested on the page, we could use a try and except import for now to solve this issue

```ruby
>>> try:
...     from osgeo import gdal
... except ImportError:
...     import gdal
```

**Comment by Leguark** [comment]
oh wow thank you for noticing this. Somehow in my environment, still works with the old way. Probably this is why travis has been failing a couple of weeks

**Comment by AlexanderJuestel** [comment]
Will be fixed with #640

**Comment by AlexanderJuestel** [comment]
I can reproduce the error in a fresh environment installing only jupyter (via conda-forge) and GemPy via pip. Looking at the requirements and setup.py, gdal will not be downloaded by default, hence it is missing when importing GemPy I think. We should therefore add gdal to the requirements or actually install it via geopandas as it will be properly installed through this package!
